docs/content/docs/testing/tinylicious.md

```diff
@@ -1,17 +1,92 @@
 ---
 title: Tinylicious
 menuPosition: 1
-status: unwritten
+editor: sdeshpande3
 ---
 
 ## What is Tinylicious?
 
-Our local service used for prototyping
-
-Link to repo
+Tinylicious is a local, in-memory service used for prototyping and development purpose. You can find the code for the [Tinylicious service](https://github.com/microsoft/FluidFramework/tree/main/server/tinylicious) and [Tinylicious client](https://github.com/microsoft/FluidFramework/tree/main/packages/framework/tinylicious-client). It can be instantiated against both `AzureClient` and `TinyliciousClient` for testing purpose.
 
 ## Using Tinylicious locally
 
+You can run tinylicious locally by executing the following command,
+
+```sh
+npx tinylicious
+```
+
+By default, Tinylicious runs on port `7070`, however you can change port by navigating to `server/tinylicious` and running the below command on `Windows Powershell` by specifying the port number,
```

Comment:
They don't need to run tinylicious from the actual server/tinylicious directory in the repo. Just setting the environment variable and then running the npx command should suffice